### PR TITLE
Testkit: Make Testcontainer logs visible

### PR DESCRIPTION
**Pull request by seglo**
Testkit testcontainers feature to toggle container logs for ZooKeeper, Brokers, and Schema Registry.

Requires SLF4J.

### CODE REVIEW

**Comment by seglo** [comment]
Logging sample https://pastebin.com/WF3JHG4z

**Comment by ennru** [comment]
Looks useful, great.

**Comment by seglo** [comment]
Thanks. I'm waiting for a failing build step to see how it looks.

File: tests/src/it/resources/reference.conf [hunk 1]
```diff
@@ -18,6 +18,8 @@ akka {
     single-expect-default = 10s
   }
 
+  kafka.testkit.testcontainers.container-logging = true
```
Comment (ennru) [inline tests/src/it/resources/reference.conf:21]: I'm a bit surprised we use `reference.conf` in IT tests, shouldn't this be `application.conf`?

**Review by ennru — APPROVED** [review]
LGTM.

File: tests/src/it/resources/reference.conf (hunk 1 above)
Comment (seglo) [reply tests/src/it/resources/reference.conf:21]: That confused me as well.  AFAIC it should be `application.conf`.

**Comment by ennru** [comment]
This has been backported to the `release-2.0.x` branch to be part of 2.0.6.

**Pull request merged**